### PR TITLE
Add foot and last foot commands to longtables

### PR DESCRIPTION
**Pull request by JarrahGos**
Previously the "\endfoot" and "\endlastfoot" commands for longtables
had to be entered manually. This commit adds python function to use
these commands.

Add longtable foot commands to change log

### CODE REVIEW

**Comment by JelteF** [comment]
Could you merge master and fix any tests that still fail then?

File: pylatex/table.py [hunk 1]
```diff
@@ -466,6 +468,28 @@ def end_table_header(self):
...
+    def end_foot(self):
+        r"""End the table foot which will appear on every page."""
+
+        if self.header:
+            msg = "Table already has a foot"
+            raise TableError(msg)
+
+        self.foot = True
+
+        self.append(NoEscape(r'\endfoot'))
```
Comment (JelteF) [inline pylatex/table.py]: Can you use Command here instead of NoEscape?

File: pylatex/table.py [hunk 2]
```diff
@@ -466,6 +468,28 @@ def end_table_header(self):
 
         self.append(NoEscape(r'\endhead'))
 
+    def end_foot(self):
+        r"""End the table foot which will appear on every page."""
+
+        if self.header:
```
Comment (JelteF) [inline pylatex/table.py]: Guess this should be self.foot

File: pylatex/table.py [hunk 3]
```diff
@@ -466,6 +468,28 @@ def end_table_header(self):
...
+            raise TableError(msg)
+
+        self.foot = True
+
+        self.append(NoEscape(r'\endfoot'))
+
+    def end_last_foot(self):
+        r"""End the table foot which will appear on the last page"""
+
+        if self.header:
```
Comment (JelteF) [inline pylatex/table.py]: self.lastFoot

File: pylatex/table.py [hunk 4]
```diff
@@ -466,6 +468,28 @@ def end_table_header(self):
...
+            msg = "Table already has a last foot"
+            raise TableError(msg)
+
+        self.lastFoot = True
+
+        self.append(NoEscape(r'\endlastfoot'))
```
Comment (JelteF) [inline pylatex/table.py]: Here also `Command` instead of `NoEscape`

**Comment by JelteF** [comment]
Thanks for the pull request, if you fix the comments I'll merge it.

**Comment by JarrahGos** [comment]
Cheers for the review. I'll rebase on master and fix those tomorrow morning and send it through. Tests should work fine then too.

**Comment by JelteF** [comment]
I was about to merge this but then the code coverage thing popped up. If you don't mind too much could you add an example for these commands? If not let me know, I'll just merge it.

**Comment by JarrahGos** [comment]
Good point. I missed that one because there wasn't an example already for longtable.

**Comment by JelteF** [comment]
Thanks for all the changes, looks good now!

**Pull request merged**